.circleci/config.yml

```diff
@@ -97,6 +97,11 @@ jobs:
             rm -rf cpython/{build,downloads}
             rm -rf emsdk/emsdk/binaryen
 
+      - run:
+          name: Zip build directory
+          command: |
+            tar cjf pyodide-build.tar.bz2  build
```

Comment:
If I remember correctly bz2 takes a long time to compress, maybe better to use gz instead? Let's see what CI says.

Reply:
50 seconds on the build-packages job.

Reply:
Indeed only 30 seconds to gzip. gzipped archive is exactly the same size as the bzipeped one.